packages/@aws-cdk/aws-backup/lib/vault.ts

```diff
@@ -0,0 +1,202 @@
+import iam = require('@aws-cdk/aws-iam');
+import kms = require('@aws-cdk/aws-kms');
+import sns = require('@aws-cdk/aws-sns');
+import { Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnBackupVault } from './backup.generated';
+
+/**
+ * Represents a Backup vault.
+ */
+export interface IBackupVault extends IResource {
```

Comment:
I would keep only `backupVaultName` in the `IBackupVault` interface. It's the only property required to deal with a (imported) vault. This will simplify import: you'll only need a `fromBackupVaultName()`